stylesheets/_component.range-slider.scss

```diff
@@ -13,12 +13,12 @@
 
     // Slider thumb
     &::-webkit-slider-thumb {
-        margin-top: -5px;
+        margin-top: -7px;
         appearance: none;
         background: color(primary);
         border: 0;
         border-radius: 50%;
-        cursor: pointer;
+        cursor: ew-resize;
```

Comment:
Is this appropriate? I'd expect `grab` perhaps?

Reply:
I think I'll revert this. Was trying to highlight that the thumb could be moved left and right, though that cursor is obviously more relevant to resizing things.

Reply:
I've changed the cursor to grab @Stanton